debian/syncd.install

```diff
@@ -1,3 +1,4 @@
 usr/bin/saidump
 usr/bin/saiplayer
 usr/bin/syncd*
+syncd/usr/bin/* usr/bin
```

Comment:
Yes. `usr/bin/syncd*` installs the standard files that are copied to `debian/tmp/usr/bin/syncd*`. My new files are in syncd/usr/bin/ and do not get copied to `debian/tmp/usr/bin/`.